content/api/entity-reference/peer-to-peer/index.html

```diff
@@ -2,15 +2,15 @@
 layout: layout-container
 name: Peer to Peer
 order: 250
-published: false
-showInNav: false
+published: true
+showInNav: true
 showBreadcrumbs: true
 ---
 
 <div ng-app="entityReferenceApp">
   <bb-entity-reference
-    api-title="Peer to Peer"
-    last-updated-date="March 14, 2017"
+    api-title="Peer-to-Peer (Beta)"
+    last-updated-date="March 28, 2017"
```

Comment:
Just as a sidenote, I'm going to look into a way to get this to be dynamically determined from the swagger.json when I get a chance (hopefully later this week). I'm thinking having a "version" property in the swagger that can be used to determine the update date (since our versions are date driven).